clipper_admin/clipper_admin/sample_config.yaml

```diff
@@ -0,0 +1,23 @@
+Model1:
```

Comment:
In addition to specifying applications and models for deployment, we want to be able to specify links between models and applications. Define a yaml dictionary (or equivalent) called "Links" whose key-value entries are defined as follows:

* `key = <APP_NAME>`
* `value = <MODEL_NAME>`

Reply:
Let's be more explicit with the links. Instead of making key value pairs let's structure the links as a list of objects, where each object has a `model` key and an `app` key. For example:

```
links:
  - model: <MODEL_NAME>
     app: <APP_NAME
  - model: <OTHER_MODEL_NAME>
     app: <OTHER_APP_NAME>
```